config/manager/manager.yaml

```diff
@@ -70,7 +70,7 @@ spec:
         - /manager
         args:
         - --leader-elect
-        image: quay.io/opendatahub/model-registry-operator:latest
+        image: controller
```

Comment:
I'm wondering however if the images are placed in env file, what is the strategy for the `config/manager/kustomization.yaml` file, please? 
I mean wouldn't that operate some changes which now should pertain to the env file?
(open question)

Reply:
Moved odh config to a separate overlay so the default kustomize image override will still work.